tensorflow/core/platform/default/logging.cc

```diff
@@ -117,6 +127,8 @@ LogMessage::~LogMessage() {
   if (TF_PREDICT_TRUE(severity_ >= min_log_level)) GenerateLogMessage();
 }
 
+int LogMessage::min_lvl_ = MinVLogLevel();
```

Comment:
Good suggestion, this has better suggestion comparing to static member variable. Will address.